yarn.lock

```diff
@@ -1107,6 +1107,13 @@
     reflect-metadata "^0.1.13"
     whatwg-fetch "^3.1.0"
 
+"@project-sunbird/discussions-ui-v8@2.0.32-beta.1":
```

Comment:
make sure which library to use. @project-sunbird or @sunbird-cb ?
@arunpilli21 @SuvajitTarento